App.js

```diff
@@ -8,91 +8,58 @@ import {
 } from 'react-native'
 
 import PaddingTop from './App/Containers/PaddingTop'
+import WeatherBox from './App/Containers/WeatherBox'
 
 
 export default class App extends React.Component {
   constructor(props) {
     super(props)
   }
 
-  outputCubes = (obj) => {
-
-    test = (num) => {
-      switch(num){
-        case(1):
-          return 'Mon'
-          break;
-        case(2):
-          return 'Tue'
-          break;
-        case(3):
-          return 'Wed'
-          break;
-        case(4):
-          return 'Thu'
-          break;
-        case(5):
-          return 'Fri'
-          break;
-        case(6):
-          return 'Sat'
-          break;
-        case(7):
-          return 'Sun'
-          break;
-        default:
-          return '?'
-          break;
-      }
-    }
```

Comment:
removed test() function and replaced with a 2D Array. Turns it into a O(1) complexity instead of O(n)